groot/attack.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+class AttackWrapper:
+    def attack_feasibility(self, X, y, order=np.inf, epsilon=0.0, options={}):
+        X_distances = self.attack_distance(X, y, order, options)
+        return X_distances < epsilon
```

Comment:
Not sure what this actually returns, I think I would expect a percentage here

Reply:
I added some docstrings 👍